pkg/cmd/server/api/v1/types.go

```diff
@@ -267,8 +267,10 @@ type MasterConfig struct {
 	// AuditConfig holds information related to auditing capabilities.
 	AuditConfig AuditConfig `json:"auditConfig"`
 
-	// EnableTemplateServiceBroker is a temporary switch which enables TemplateServiceBroker.
-	EnableTemplateServiceBroker bool `json:"enableTemplateServiceBroker"`
+	// TemplateServiceBrokerConfig holds information related to the template
+	// service broker.  The broker is enabled if TemplateServiceBrokerConfig is
+	// non-nil.
+	TemplateServiceBrokerConfig *TemplateServiceBrokerConfig `json:"templateServiceBrokerConfig"`
```

Comment:
i'd prefer an explicit enabled/disabled field, mainly so that I don't have to erase all my configuration just to disable the broker temporarily (or re-add all my configuration to re-enable it).  I'm sure there are many opinions on that, but I just wanted to voice mine...